crates/server/src/store/blocklist.rs

```diff
@@ -95,8 +99,45 @@ impl BlocklistAuthority {
             block_message: config.block_message.clone(),
             consult_action: config.consult_action,
             log_clients: config.log_clients,
+            #[cfg(feature = "metrics")]
+            metrics: BlocklistMetrics {
+                entries: gauge!("blocklist_list_entries"),
+                blocked_queries: counter!("blocklist_blocked_queries_total"),
+                logged_queries: counter!("blocklist_logged_queries_total"),
+                total_hits: counter!("blocklist_list_hits_total"),
+                total_queries: counter!("blocklist_queries_total"),
+            },
         };
 
+        #[cfg(feature = "metrics")]
+        {
+            describe_gauge!(
+                "blocklist_list_entries",
+                Unit::Count,
+                "The total number of entries in all configured blocklists",
+            );
+            describe_counter!(
+                "blocklist_blocked_queries_total",
+                Unit::Count,
+                "The total number of requests that were blocked by the blocklist authority",
+            );
+            describe_counter!(
+                "blocklist_logged_queries_total",
+                Unit::Count,
+                "The total number of requests that were logged by the blocklist authority.",
```

Comment:
nit: this desc has a trailing `.` and the others do not. Can you drop this instance, or make the others consistent?